frontend/packages/knative-plugin/src/components/test-function/ResponsePane.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core/dist/esm/components/Alert';
+import { Label } from '@patternfly/react-core/dist/esm/components/Label';
+import { FormikProps, FormikValues } from 'formik/dist/types';
+import { useTranslation } from 'react-i18next';
+import { ExpandCollapse, Loading } from '@console/internal/components/utils';
+import { CodeEditorField } from '@console/shared/src/components';
+
+import './TestFunctionModal.scss';
+
+type ResponseProps = {
+  status: string;
+  statusCode: number;
+  header: Record<string, string[]>;
+  body: string;
+};
+
+const ResponsePane: React.FC<FormikProps<FormikValues>> = ({ values }) => {
+  const { t } = useTranslation();
+  const { statusCode, header, body } = values.response as ResponseProps;
```

Comment:
You can define a custom type for the FormikValues:

```suggestion
type Response = {
  status: string;
  statusCode: number;
  header: Record<string, string[]>;
  body: string;
};

type ResponsePaneFormikValues = {
  response: Response | undefined;
}

const ResponsePane: React.FC<FormikProps<ResponsePaneFormikValues>> = ({ values }) => {
  const { t } = useTranslation();
  const { statusCode, header, body } = values.response;
```

Reply:
Done :)